tests/xtro-sharpie/common-CoreImage.ignore

```diff
@@ -233,3 +233,15 @@
 !missing-selector! +CIFilter::torusLensDistortionFilter not bound
 !missing-selector! +CIFilter::twirlDistortionFilter not bound
 !missing-selector! +CIFilter::vortexDistortionFilter not bound
+## xcode 13
+!missing-selector! +CIFilter::convolutionRGB3X3Filter not bound
+!missing-selector! +CIFilter::convolutionRGB5X5Filter not bound
+!missing-selector! +CIFilter::convolutionRGB7X7Filter not bound
+!missing-selector! +CIFilter::convolutionRGB9HorizontalFilter not bound
+!missing-selector! +CIFilter::convolutionRGB9VerticalFilter not bound
+!missing-selector! +CIFilter::linearLightBlendModeFilter not bound
+!missing-selector! +CIFilter::personSegmentationFilter not bound
+!missing-selector! +CIFilter::vividLightBlendModeFilter not bound
```

Comment:
why were those not bound ?

Reply:
implemented these filters as other filters were in the past, so as per the comment at the top of the file:
`## we already provide alternative API to create the filters`